src/components/common/Stepper.vue

```diff
@@ -177,14 +182,58 @@ export default class Stepper extends Vue {
 @import '@/assets/styles/theme.scss';
 
 #step-buttons-container {
+  display: flex;
+  justify-content: space-evenly;
+  margin: 0;
+  padding: 2rem 0;
   background: $BCgovInputBG;
 }
 
-.mdi-check-circle {
+.step {
+  display: flex;
+  flex-direction: column;
+  flex: 1 1 auto;
+  align-items: center;
+  justify-content: center;
+}
+
+.step__indicator {
+  position: relative;
+  width: 100%;
+  text-align: center;
+}
+
+.step__line {
+  position: absolute;
+  top: 50%;
+  left: 0;
+  height: 2px;
```

Comment:
Just curious why we are using px here and rem in other spots?

Reply:
This is the height of the connector line. It never needs to be scaled, which is the usual reason to use "root ems". 1 px seems too thin to me and 3 px seems too thick.